scripts/makeLegacyPlots.py

```diff
@@ -719,6 +727,13 @@ def scaleStatSystUncertaintyBandForStack (grUncert,bkgSum):
     for n in hDatas: print "Integral ", hDatas[n].GetName(), " : ", hDatas[n].Integral(), " - ", hDatas[n].Integral(-1,-1)
     for i, name in enumerate (sigNameList): print "Integral ", hSigs[sigList[i]].GetName(), " : ", hSigs[sigList[i]].Integral(), " - ", hSigs[sigList[i]].Integral(-1,-1)
     
+    # Store yields in a txt file for reference
+    with open('./LegacyPlots/Legacy' + args.year + '/' + args.channel + '/' + args.sel+'/yields_'+args.var+'_'+args.sel+'_'+args.reg+'.txt','w') as yields_file:
+        yields_file.write('=== Legacy' + args.year + '/' + args.channel + '/' + args.sel+'/'+args.reg+'/'+args.var+' ===\n')
+        for h in hBkgList: yields_file.write("Integral: "+h.GetName()+" : "+str(h.Integral())+" - "+str(h.Integral(-1,-1))+"\n")
+        for n in hDatas  : yields_file.write("Integral: "+hDatas[n].GetName()+" : "+str(hDatas[n].Integral())+" - "+str(hDatas[n].Integral(-1,-1))+"\n")
+        for i, name in enumerate (sigNameList): yields_file.write("Integral: "+hSigs[sigList[i]].GetName()+" : "+str(hSigs[sigList[i]].Integral())+" - "+str(hSigs[sigList[i]].Integral(-1,-1))+"\n")
+    yields_file.close()
 
     #################### PERFORM DIVISION BY BIN WIDTH AND DO NECESSARY STACKS #######################
     bkgStackNS = makeStack ("bkgStackNS", hBkgList)
```

Comment:
here create a bkg sum non scaled together with the other non scaled things
`bkgSumNS = makeSum ("bkgSumNS", hBkgList)`